conda/recipes/cuml/build.sh

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
 
 # This assumes the script is executed from the root of the repo directory
-./build.sh cuml
-$PYTHON -c 'import cuml'
```

Comment:
why remove the import test?

Reply:
This was not a test. It was a bad attempt at solving a bug a few months ago that I forgot to remove, so it really does nothing here